### PR TITLE
Fix format for tag

### LINKED ISSUE #28: Cannot format grammar rules while tags exist
## **Describe the bug**

When trying to format a pest file with tags in it, I received an error message and the file remained unchanged.

## **To Reproduce**

1. Create an empty grammar file, let's say `grammar.pest`, and type in these rules:
    ```
    some_rule = { #tag = "test" ~ "rule" ~ "one" }
    another_rule_with_a_long_name = { "test" ~ "rule" ~ "two" ~ "with" ~ "a" ~ "long" ~ "name" }
    ```
3. Run `pestfmt grammar.pest` and you'll get an error message, indicating that pestfmt is expecting expression or COMMENT at the position of  `#tag`
4. Remove the tag in `some_rule`, so the content of `grammar.pest` is as follows:
    ```
    some_rule = { "test" ~ "rule" ~ "one" }
    another_rule_with_a_long_name = { "test" ~ "rule" ~ "two" ~ "with" ~ "a" ~ "long" ~ "name" }
    ```
3. Run `pestfmt grammar.pest` again and the error message disappears

### **Reproduction File**

The grammar rule file is described as above.

## **Expected behavior**

The "Format Document" functions normally, whether there are tags in my rules or not.

## **Environment (please complete the following information):**

 - OS: Windows 11 23H2
 - pestfmt version: 0.2.4

### PR DESCRIPTION
**Pull request by huacnlee**
## Summary by CodeRabbit

- **New Features**
	- Add `make update_grammar` command for update grammar rules.
- **Bug Fixes**
	- Fix to support `tag` rule #28 
- **Documentation**
	- Add development guide.

### CODE REVIEW

**Review by tomtau — APPROVED** [review]

**Comment by tomtau** [comment]
thanks!

File: DEVELOPMENT [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## Development Guide
+
+- Run `make test` to test all.
+
+## Update Pest original `grammar.pest`.
+
+The `src/grammar.pest` file is a copy of the original
+[grammar.pest](https://github.com/pest-parser/pest/blob/master/meta/src/grammar.pest) file from the Pest repository.
+
+To update it, run the following command:
```
Comment (NoahTheDuke) [inline DEVELOPMENT:10]: No command is written here.

File: DEVELOPMENT [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+But it have a little bit different from the original file:
```
Comment (NoahTheDuke) [inline DEVELOPMENT:12]:
```suggestion
But it is a little bit different from the original file:
```

**Review by NoahTheDuke — COMMENTED** [review]
I see that you're already merged but these stood out to me

**Pull request merged**